pkg/logging/yaml_parser.go

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) The Thanos Authors.
+// Licensed under the Apache License 2.0.
+
+package logging
+
+type ReqLogConfig struct {
+	HTTP    ProtocolConfigs `yaml:"http"`
```

Comment:
I am surprised those have the same type 😱

Reply:
I have broken down this into HTTP and gRPC configs, should be readable now